csaf/vex/cve/2024/cve-2024-6601.json

```diff
@@ -11,14 +11,24 @@
     "tracking": {
       "id": "CVE-2024-6601",
       "initial_release_date": "2024-07-30T14:59:26Z",
-      "current_release_date": "2024-07-30T14:59:26Z",
+      "current_release_date": "2024-07-30T15:07:04Z",
       "status": "final",
-      "version": "1",
+      "version": "3",
```

Comment:
The only "weird" part of this is the jump from V1 to V2 ... I'm gonna assume you did this by doing the x86_64 and aarch64 on different invocations of `csaf-gen.py`